Libraries/JavaScriptAppEngine/System/JSTimers/JSTimersExecution.js

```diff
@@ -70,6 +73,12 @@ var JSTimersExecution = {
       } else if (type === JSTimersExecution.Type.requestAnimationFrame) {
         var currentTime = performanceNow();
         callback(currentTime);
+      } else if (type === JSTimersExecution.Type.requestIdleCallback) {
+        callback({
+          timeRemaining: function() {
+            return Math.max(0, 17 - (performanceNow() - frameTime));
```

Comment:
Yeah I dont think we should implement this in this PR either, as for remote debugging we can probably just skip this check when debugging in chrome and sync calls are not possible as it is more of an optimisation than something that is required.